src/app/devices/sol/sol.component.html

```diff
@@ -1,4 +1,9 @@
-<app-device-toolbar [deviceState]="deviceState" (stopSol)="stopSol()" [isLoading]="isLoading"></app-device-toolbar>
-<div class="container">
-    <div id="terminal" class="xtermDisplay" style="width: fit-content;"></div>
-</div>
\ No newline at end of file
+<app-device-toolbar
+  [deviceState]="deviceState"
+  (stopSol)="stopSol()"
+  [isLoading]="isLoading"
+></app-device-toolbar>
+<amt-sol  *ngIf="readyToLoadSol"
```

Comment:
missing deviceId, authtoken, mpsServer -- please update sol to have these passed in

Reply:
added deviceId, authtoken, mpsServer